src/sentry/integrations/github/integration.py

```diff
@@ -120,6 +120,12 @@
         """,
         IntegrationFeatures.TICKET_RULES,
     ),
+    FeatureDescription(
+        """
+        Connect multiple Sentry organizations to a single GitHub account.
+        """,
+        IntegrationFeatures.SCM_MULTI_ORG,
+    ),
```

Comment:
do we usually feature flag adding this feature description? I was looking at some other PRs that added `FeatureDescriptions` and they weren't ever feature flagged (??)

Reply:
removed listing the feature in the integration page, we'll do this at GA time